website/client/tests/unit/libs/highlightUsers.spec.js

```diff
@@ -17,6 +17,24 @@ describe('highlightUserAndEmail', () => {
     expect(result).to.contain('<span class="at-text at-highlight">@user</span>');
   });
 
+  it('highlights username sandwiched with underscores', () => {
+    const text = 'hello @_user_';
+
+    const result = highlightUsers(text, '_user_', 'displayedUser');
+    expect(result).to.contain('<span class="at-text at-highlight">@_user_</span>');
+    expect(result).to.not.contain('<em>');
+    expect(result).to.not.contain('</em>');
+  });
+
+  it('highlights username sandwiched with double underscores', () => {
+    const text = 'hello @__user__';
+
+    const result = highlightUsers(text, '__user__', 'displayedUser');
+    expect(result).to.contain('<span class="at-text at-highlight">@__user__</span>');
```

Comment:
I would let one of these test cases cover the other branch (The highlighted user is not the logged in user) as to cover more variance in the test cases.

```suggestion
    const result = highlightUsers(text, 'differentUser', 'displayedDifferentUser');
    expect(result).to.contain('<span class="at-text">@__user__</span>');
```